projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-site-migration-wpcom-migration-key.php

```diff
@@ -73,10 +73,6 @@ public function can_access() {
 			return false;
 		}
 
-		if ( 'read' === get_option( $this->key_is_read_option_name, false ) ) {
```

Comment:
It looks like the `key_is_read_option_name` option is not used anymore. What do you think of cleaning up the related logic (removing the variable and the update_option call)?

Reply:
I was thinking that it can be useful to keep this data around maybe? 🤔 Just in case we want to bring this logic back or use it elsewhere?

Reply:
Makes sense, removed it here https://github.com/Automattic/jetpack/pull/40270/commits/2871fce298dd39858f6dc7298ca51ec8e973affe